recipes/email/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "emails" %}
+{% set version = "0.5.15" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fcc02567a528eae6b66d2a5c20ce7a0326e4f6b201bc8ae302f89413164db06a
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=2.6
+    - pip
+  run:
+    - python >=2.6
+    - chardet
+    - cssutils
+    - lxml
+    - premailer
+    - python-dateutil
+    - requests
+
+test:
+  imports:
+    - emails
```

Comment:
Add imports of submodules (e.g. `emails.store`, `emails.backend`, etc)

Reply:
why?

Reply:
the author specifies what the top level exposes, who am I to assume the validity of submodule imports https://github.com/lavr/python-emails/blob/master/emails/__init__.py#L48

Reply:
or are you just trying to get it to match the setup.py? https://github.com/lavr/python-emails/blob/master/setup.py#L116

Reply:
Part of it is matching the `setup.py`, part of it is just to extend the check to make sure that there aren't missing dependencies that aren't imported in a limited top level `__init__.py`

Reply:
ill add them, but I haven't done this for any other libraries recently

Reply:
I guess the reviewers aren't uniform in what they are looking for. I tend to check for that sort of thing, but some don't.

Reply:
no worries, easy to add.